src/normalizers/keywordTranslatorMaker.js

```diff
@@ -0,0 +1,31 @@
+import { regexMatchingDecoderMaker, regexMatchingEncoderMaker } from './regex.js';
+
+export function keywordTranslatorMaker (name, keywordMap) {
+    if (keywordMap) {
```

Comment:
maybe if you check that this is an object with at least one key, you don't need the `NO_PROTECTED_STRINGS` bit

Reply:
Makes sense.  Changed.